conductr_cli/sandbox_run_jvm.py

```diff
@@ -83,36 +85,60 @@ def run(args, features):
     return SandboxRunResult(core_pids, core_addrs, agent_pids, agent_addrs)
 
 
-def log_run_attempt(args, run_result, is_started, wait_timeout):
+def log_run_attempt(args, run_result, feature_results, is_conductr_started, is_proxy_started, wait_timeout):
     """
     Logs the run attempt. This method will be called after the completion of run method and when all the features has
     been started.
 
     :param args: args parsed from the input arguments
     :param run_result: the result from calling sandbox_run_jvm.run() - instance of sandbox_run_jvm.SandboxRunResult
-    :param is_started: sets to true if sandbox is started
+    :param feature_results: the feature result
+    :param is_conductr_started: sets to true if sandbox is started
+    :param is_proxy_started: sets to true if proxy is started
     :param wait_timeout: the amount of timeout waiting for sandbox to be started
     :return:
     """
     log = logging.getLogger(__name__)
     if not args.no_wait:
-        if is_started:
-            log.info(headline('Summary'))
+        if is_conductr_started:
+            log.info(h1('Summary'))
+            log.info(h2('ConductR'))
             log.info('ConductR has been started:')
-
-            nr_instance_core = len(run_result.core_pids)
-            plural_core = 's' if nr_instance_core > 1 else ''
-            log.info('  core: {} instance{}'.format(nr_instance_core, plural_core))
-
-            nr_instance_agent = len(run_result.agent_pids)
-            plural_agents = 's' if nr_instance_agent > 1 else ''
-            log.info('  agent: {} instance{}'.format(nr_instance_agent, plural_agents))
-
-            log.info('Check current bundle status with:')
+            plural_core = 's' if len(run_result.core_addrs) > 1 else ''
+            log.info('  core instance{} on {}'.format(plural_core, ', '.join(str(i) for i in run_result.core_addrs)))
+            plural_agent = 's' if len(run_result.agent_addrs) > 1 else ''
+            log.info('  agent instance{} on {}'.format(plural_agent, ', '.join(str(i) for i in run_result.agent_addrs)))
+            log.info('ConductR service locator has been started on:')
+            log.info('  {}:{}'.format(run_result.host, DEFAULT_SERVICE_LOCATOR_PORT))
+
+            log.info(h2('Proxy'))
+            if is_proxy_started:
+                log.info('HAProxy has been started')
+                log.info('Your Bundles are by default accessible on:')
+                log.info('  {}:{}'.format(run_result.host, CONDUCTR_BUNDLE_PROXY_PORT))
+            else:
+                log.info('HAProxy has not been started')
+                log.info('Start the proxy by installing and running Docker')
```

Comment:
This may be interpreted as when Docker is installed and running, the HAProxy will be automatically available after `sandbox run`.

Perhaps instead say "To enable proxying ensure Docker is running and restart the sandbox".